src/Pate/MemCell.hs

```diff
@@ -97,6 +100,22 @@ traverseWithCell ::
 traverseWithCell (MemCellPred memPred) f =
   MemCellPred <$> Map.traverseWithKey (\(Some cell@MemCell{}) p -> f cell p) memPred
 
+rebuild ::
```

Comment:
This was just a cheap way to avoid a module import cycle, since the Location module depends on MemCell. I'll give it the same name at least so it's clear what's going on.